### PR TITLE
voting slider issue

### PR DESCRIPTION
**Pull request by MTSaeed**
**What does this PR?**
voting slider range should has a difference of 5%.

**Steps to reproduce**
open any post and try to vote that post.
observe that the difference of voting slider should be 5.

**Issue number**
fixes #[1083](https://github.com/ecency/ecency-vision/issues/1083)

**Screenshots/Video**

https://user-images.githubusercontent.com/106739598/199215883-6d88cbee-6dcc-4967-b6d4-bd0e2d742580.mp4

### CODE REVIEW

**Comment by feruzm** [comment]
@MTSaeed This seem to lock you in 5% range always, idea is to allow in between also but also easier to select values, someone suggested visual separator every 5 or 10% but allow easy scrolling in between as well. Is it possible?

**Comment by MTSaeed** [comment]
@feruzm kindly check this.

https://user-images.githubusercontent.com/106739598/201622131-4b7f2f0a-395b-4f8d-94ad-5cca266b0b76.mp4

**Comment by feruzm** [comment]
@MTSaeed much better, can we make it like this, meaning have small dot separators inside slider and on desktop show 10 points (0 10 20 30 40 ... 90 100) but on mobile view show 5 points (0 25 50 75 100).

and we can slightly increase length of slider so numbers won't collide to each other, see last screenshot.

Screenshot 2022-11-14 at 15 31 07

Screenshot 2022-11-14 at 15 34 31

image

**Comment by MTSaeed** [comment]
Loom video link

https://www.loom.com/share/fb82632df2a54927b03844c0478a90a1

File: src/common/components/entry-vote-btn/_index.scss [hunk 1]
```diff
@@ -464,3 +465,55 @@ p {
...
+  flex-direction: row;
+  flex: 1;
+  justify-content: flex-start;
+}
+
+.options p {
+    font-size: 15px;
+}
+
+.line-slider{
```
Comment (feruzm) [inline src/common/components/entry-vote-btn/_index.scss:490]: @MTSaeed As we discussed in our call, there are couple issues still remaining, here in this picture I clicked 75 but got this...
image

**Review by feruzm — CHANGES_REQUESTED** [review]

**Review by feruzm — APPROVED** [review]
lgtm

**Pull request merged**